src/App.tsx

```diff
@@ -0,0 +1,19 @@
+import { Box, Sky } from "@react-three/drei"
+import { Canvas } from "@react-three/fiber"
+
+function App() {
+
```

Comment:
Maybe you can start to comment also with easy lines of code for a better overview ;)

Reply:
Next time :)